rules-reviewed/quarkus/springboot/springboot-web-to-quarkus.windup.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="springboot-web-to-quarkus"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset gives hints to migrate from Spring Web to Quarkus spring-web extension
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final" />
+        </dependencies>
+        <sourceTechnology id="springboot" />
+        <targetTechnology id="quarkus" />
+    </metadata>
+    <rules>
+        <rule id="springboot-web-to-quarkus-00000">
+            <when>
+                <dependency groupId="org.springframework" artifactId="spring-web"/>
+            </when>
+            <perform>
+                <hint title="Replace the SpringBoot Web artifact with Quarkus 'spring-web' extension" effort="1" category-id="mandatory">
+                    <message>
+                        Replace the Spring Web artifact with Quarkus `spring-web` extension
+
+                        Spring Web is a spring-web artifact brought transitively by any `org.springframework:spring-web*` dependency
```

Comment:
If you want to add a new line after "dependency" you should add 2 spaces at the end
```suggestion
                        Spring Web is a spring-web artifact brought transitively by any `org.springframework:spring-web*` dependency  
```